src/views/RecIM/views/Activity/Activity.module.scss

```diff
@@ -99,6 +99,32 @@
   background-color: $alternative-red;
 }
 
+.logoFlexBox {
+  flex-direction: column;
+  display: inherit;
+}
+
+.logo {
+  border-radius: 50%;
+  width: 6.9em;
+  height: 6.9em;
+  // max-height: 100px;
```

Comment:
```suggestion

```

Reply:
This enforces the `max-height` of the Logo to 100px. This will not affect the aspect ratio if the height is <= 100px when width = 100px. The substitute to this styling is to set the aspect ratio on Cropper to default 1, so we don't have to care about the height anymore.